### PR TITLE
Moving Functions

### PR DESCRIPTION
**Pull request by cslotboom**
Here I move around some functions and imports. This will make it easier to track final changes to all the functions.

All of the functions  from "Get_Fiber Response" and "Get_Rendering_Animations" has been moved, and the files deleted.

Some functions will need to be imported now. I will fix it as I go through my final cleaning and making the test suite.

### CODE REVIEW

File: openseespy-pip/openseespy/postprocessing/Get_Rendering.py [hunk 1]
```diff
@@ -792,3 +798,618 @@ def nodecoordsFinal(nodetag):
...
+    # Get nodes and elements
+    ndm = len(nodes[0,1:])
+    Nnodes = len(nodes[:,0])
+    Nele = len(elements)
+    
+    nodeLabels = nodes[:, 0]       
+    NodeText = [None]*Nnodes
+
+    # initialize figure
+    fig, ax = _initializeFig(nodes[:,1:], ndm)
```
Comment (u-anurag) [inline openseespy-pip/openseespy/postprocessing/Get_Rendering.py]: _initializeFig needs a link to the internal_plotting_functions, i.e. ipltf._initializeFig()

File: openseespy-pip/openseespy/postprocessing/Get_Rendering.py [hunk 2]
```diff
@@ -792,3 +798,618 @@ def nodecoordsFinal(nodetag):
...
+    Nele = len(elements)
+    
+    nodeLabels = nodes[:, 0]       
+    NodeText = [None]*Nnodes
+
+    # initialize figure
+    fig, ax = _initializeFig(nodes[:,1:], ndm)    
+    
+	# Adjust plot area.   
+    _setStandardViewport(fig, ax, nodes[:,1:], ndm, Disp[0,:,:])
```
Comment (u-anurag) [inline openseespy-pip/openseespy/postprocessing/Get_Rendering.py]: It should be ipltf._setStandardViewport()

File: openseespy-pip/openseespy/postprocessing/Get_Rendering.py [hunk 3]
```diff
@@ -792,3 +798,618 @@ def nodecoordsFinal(nodetag):
...
+    # ========================================================================
+    
+    initialDisp = nodes[:, 1:] + Disp[0,:,:]
+    
+    # Add Text
+    if ndm == 2:
+        time_text = ax.text(0.95, 0.01, '', verticalalignment='bottom', 
+                            horizontalalignment='right', transform=ax.transAxes, color='grey')
+        
+        EQObjects = _plotEle_2D(nodes, elements, initialDisp, fig, ax, show_element_tags)
```
Comment (u-anurag) [inline openseespy-pip/openseespy/postprocessing/Get_Rendering.py]: ipltf._plotEle_2D()

File: openseespy-pip/openseespy/postprocessing/Get_Rendering.py [hunk 4]
```diff
@@ -792,3 +798,618 @@ def nodecoordsFinal(nodetag):
...
+        [EqfigLines, EqfigSurfaces, EqfigText] = EQObjects 
+        EqfigNodes, = ax.plot(tempDisp[0,:,0],tempDisp[0,:,1], **node_style)  
+    
+        if show_node_tags == 'yes':
+            for j in range(Nnodes):
+                NodeText[j] = ax.text(*nodes[j,1:]*1.02, str(int(nodes[j,0])), **node_text_style)
+                
+    if ndm == 3:
+        
+        EQObjects = _plotEle_3D(nodes, elements, initialDisp, fig, ax, show_element_tags)
```
Comment (u-anurag) [inline openseespy-pip/openseespy/postprocessing/Get_Rendering.py]: ipltf._plotEle3D()

File: openseespy-pip/openseespy/postprocessing/Get_Rendering.py [hunk 5]
```diff
@@ -792,3 +798,618 @@ def nodecoordsFinal(nodetag):
...
+    # Process Node information, Calulate number of degrees of freedom
+    nodeList = nodes[:,0]    
+    Nnodes = len(nodeList)
+    nodeCoordArray = nodes[:,1:]
+    ndm = len(nodes[0,1:])   
+
+    # Get displacements
+    if Plot_Displacements == 'yes':
+        # Get Node coordinants
+        OBD = readODB(ModelName, LoadCaseName)
```
Comment (u-anurag) [inline openseespy-pip/openseespy/postprocessing/Get_Rendering.py:1313]: Use idbf._readNodeDispData(Model,LoadCase) instead.

Comment (u-anurag) [reply openseespy-pip/openseespy/postprocessing/Get_Rendering.py:1313]: You can check plot_deformedshape() function for reference.

File: openseespy-pip/openseespy/postprocessing/Get_Rendering.py [hunk 6]
```diff
@@ -792,3 +798,618 @@ def nodecoordsFinal(nodetag):
...
+    
+    DispNodeCoordArray = nodes[:,1:] + DispNodeArray
+
+    
+    Nele = len(elements)
+    figNodeTags = [None]*Nele
+    NodeText = [None]*Nnodes
+    
+    # Initialize figure
+    fig, ax = _initializeFig(DispNodeCoordArray, ndm)
```
Comment (u-anurag) [inline openseespy-pip/openseespy/postprocessing/Get_Rendering.py]: it should be ipltf._initializeFig()

File: openseespy-pip/openseespy/postprocessing/Get_Rendering.py [hunk 7]
```diff
@@ -792,3 +798,618 @@ def nodecoordsFinal(nodetag):
...
+    
+    # Check if the model is 2D or 3D
+    if ndm == 2:
+        
+        # Plot elements
+        OutputObjects = _plotEle_2D(nodes, elements, DispNodeCoordArray, fig, ax, show_element_tags)
```
Comment (u-anurag) [inline openseespy-pip/openseespy/postprocessing/Get_Rendering.py]: ipltf._plotEle_2D()

File: openseespy-pip/openseespy/postprocessing/Get_Rendering.py [hunk 8]
```diff
@@ -792,3 +798,618 @@ def nodecoordsFinal(nodetag):
...
+        ax.set_ylabel('Y')
+			
+
+    if ndm == 3:
+        
+
+        print('3D model')
+		
+        # Plot Model and make Objects
+        OutputObjects = _plotEle_3D(nodes, elements, DispNodeCoordArray, fig, ax, show_element_tags)
```
Comment (u-anurag) [inline openseespy-pip/openseespy/postprocessing/Get_Rendering.py]: ipltf._plotEle_3D()

File: openseespy-pip/openseespy/postprocessing/internal_database_functions.py [hunk 9]
```diff
@@ -293,3 +293,13 @@ def _readNodeDispData(ModelName,LoadCaseName):
     nodes_displacement = tempDisp
 	
     return timeSteps, nodes_displacement
+
+
+#### Read fibre data
+
+
+def recordFiberResponse(OutputName,eleNumber, sectionNumber):
```
Comment (u-anurag) [inline openseespy-pip/openseespy/postprocessing/internal_database_functions.py]: Where does it save the file? If this function is available to users, shouldn't it be in Get_Rendering() ?

**Review by u-anurag — CHANGES_REQUESTED** [review]
@cslotboom There are some comments. I assume you are running all the OpenSeesPy examples using the changed plotting files on your local machine to make sure they all work before you initiate a PR. If not, I would suggest to save all the plotting files in C-Moving_Functions branch to your local machine and run all the examples using them. That would be a better way to find all the missing links.

**Comment by cslotboom** [comment]
Hey @u-anurag - sorry I think I wasn't very clear. I was planning to make those changes once the files were moved, but I can change those first if that is your preference.

I did realize this breaks a lot of things. 
I thought it would be easier to track changes to the functions if I updated the file location first, then updated the functions. I'm happy to make the changes in this pull request, if that is easier.

**Review by u-anurag — APPROVED** [review]
Looks good. Please use idbf._readNodesandElements(ModelName) and idbf._readNodeDispData(ModelName,LoadCaseName) functions instead of readODB().

**Pull request merged**